comp/otelcol/logsagentpipeline/component.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
```

Comment:
Should this be 2024 instead ?
```suggestion
// Copyright 2024-present Datadog, Inc.
```